client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -373,62 +391,104 @@ export default React.createClass( {
 
 	unverifiedDomainsCanManage() {
 		const domains = this.getDomains().filter( domain => domain.isPendingIcannVerification && domain.currentUserCanManage );
+		const severity = domains.some( ( { registrationMoment } ) =>
+			registrationMoment && moment( registrationMoment )
+				.add( 2, 'days' )
+				.isAfter( moment() ) ) ? 'is-info' : 'is-error';
 
 		if ( domains.length === 0 ) {
 			return null;
 		}
 
 		if ( domains.length === 1 ) {
-			const domain = domains[ 0 ].name,
-				fullMessage = this.translate(
+			const domain = domains[ 0 ].name;
+			let fullMessage, compactMessage;
+			if ( severity === 'is-error' ) {
+				fullMessage = this.props.translate(
 					'Your domain {{strong}}%(domain)s{{/strong}} may be suspended because your email address is not verified.',
 					{
```

Comment:
Maybe we could extract this object above and just pass to all `translate` calls, as it's the same. :)